tests/Tests/500__jupyterhub/autoscaling-gpus.robot

```diff
@@ -0,0 +1,103 @@
+*** Settings ***
+Documentation    Tests for a scenario in which a gpu machine pool with autoscaling
+...              Is present on the cluster. Tests that the spawner shows the correct
+...              No. of GPUs available and that autoscaling can be triggered
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/GPU.resource
+Resource         ../../Resources/Page/OCPDashboard/Pods/Pods.robot
+Library          JupyterLibrary
+Suite Setup      Spawner Suite Setup
+
+
+*** Variables ***
+${NOTEBOOK_IMAGE} =         minimal-gpu
+
+
+*** Test Cases ***
+Verify Number Of Available GPUs Is Correct With Machine Autoscalers
+    [Documentation]  Verifies that the number of available GPUs in the
+    ...    spawner dropdown is correct; i.e., it should show the maximum
+    ...    number of GPUs available in a single node, also taking into account
+    ...    nodes that can be autoscaled.
+    [Tags]    Tier2    Resources-GPU-Autoscaling
+    ...       ODS-1820
+    ${maxNo} =    Find Max Number Of GPUs In One MachineSet
+    ${maxSpawner} =    Fetch Max Number Of GPUs In Spawner Page
+    Should Be Equal    ${maxSpawner}    ${maxNo}
+
+Verify Autoscale Can Be Triggered
+    [Documentation]    Tries spawning a server requesting 1 GPU, which should
+    ...    trigger a node autoscale.
+    [Tags]    Tier2    Resources-GPU-Autoscaling
+    ...       ODS-1821
+    Spawn Notebook And Trigger Autoscale
+    ${serial} =    Get GPU Serial Number
+    ${pod_node} =    Get User Server Node
+    Log    ${serial}
+    Log    ${pod_node}
+    Set Suite Variable    $NODE    ${pod_node}
```

Comment:
This Suite Variable is used in next test, making one dependent on the following one. I think it would be better to merge the tests and make `Verify Node Is Scaled Down` just a keyword